[PFX-743] - Quick Start Doc (#908)
godaddy/gasket: docs/quick-start.md

## Create

To get started, you will need to install the `create-gasket-app` package or use a package runner such as `npx` or `yarn exec`.

```bash
npm i -g create-gasket-app
# OR
npx create-gasket-app
```

> Now you can create gasket apps with the `create-gasket-app` command, the `appname` and the preset you would like to use. In this example we will create a Next.js app. The `appname` is the only required argument.

Reviewer: ```suggestion
In this example we will create a Next.js app by specifying the app name along with the preset `@gasket/preset-nextjs`:
```

```bash

> npm run build
> GASKET_ENV=local npm run start

Reviewer: ```suggestion
npm run local
```

Author: I'm not sure what the change is here. I have the local command in the section above. Do you want me to remove the entire seciton?

Reviewer: My bad... 

So, to be concise, leave out the GASKET_ENV in the example.
```
npm run build
npm run start
```

local is the default env now
